### ISSUE TITLE
Support for EasyCap 4 Channel USB DVR

### ISSUE THREAD

**Issue by zzzbatmand** [ccrisan/motioneyeos#1152]
I am aware that there is support for the "EasyCap USB Video Recorder" as shown in the issue #281, but is it possible to get support for the "EasyCap 4 Channel USB DVR" also?
I don't know if it is possible, but I would love it, if it is.

**Comment by ccrisan** [comment]
@zzzbatmand what hardware are you on? Is this a Raspberry PI? If yes, have you tried Raspbian? Does it work?

**Comment by zzzbatmand** [comment]
@ccrisan Yes, it is on a raspberry pi, sorry for not mentioning it.

I am not a professional at programming for raspberry and linux, but from what I have gathered from searching around, I have found that it is possible to use the device on raspbian, but it is problematic and not even certain that it will work.

From my perspective I think that it would be too hard to make it work, so this request can be put down if you don't want to work in it.
